tracer/src/Datadog.Trace.BenchmarkDotNet/DatadogExtensions.cs

```diff
@@ -33,7 +33,7 @@ internal static IConfig WithDatadog(this IConfig config, bool? enableProfiler)
     {
         var cfg = config.AddLogger(DatadogSessionLogger.Default);
 
-        enableProfiler ??= (EnvironmentHelpers.GetEnvironmentVariable(ConfigurationKeys.ProfilingEnabled) ?? string.Empty).ToBoolean();
+        enableProfiler ??= (EnvironmentHelpers.GetEnvironmentVariable(ConfigurationKeys.ProfilingEnabled) ?? string.Empty).ToBoolean() ?? false;
```

Comment:
If `enableProfiler` is `null`, and the `ConfigurationKeys.ProfilingEnabled` key isn't set, `ToBoolean()` returns a nullable, which we then try to unwrap on Line 47 and 💥

Reply:
Then we probably don't need the `!` at line 47 anymore?